api/rest/v2/observation.md

```diff
@@ -10,95 +10,193 @@ permalink: /api/rest/v2/observation
+| entity.expression                                     | string | [Relation expression](/api/rest/v2/index.html#relation-expressions) that represents the  entities to query.  At least one of `entity.dcids` or `entity.expression` is required.|
+| select <br /> <required-tag>Required</required-tag>  | string literal | `select=variable` and `select=entity` are required. If specifed without `select=date` and `select=value`, no observations are returned. You can use this to first check the existence of variable-entity pairs in the data and fetch all the variables that have data for given entities. |
+| select <br /> <optional-tag>Optional</optional-tag> | string literal | If used, you must specify both `select=date` and `select=value`. Returns actual observations, with the date and value for each variable and entity queried. |
+{: .doc-table }
+
+{: #date-string}
+### Date-time string formats 
+
+Here are the possible values for specifying dates/times:
+- `LATEST`: Fetch the latest observations only.
+- <var>DATE_STRING</var>: Fetch observations matching the specified date(s) and time(s). The value must be in the [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) format used by the target variable; for example, `2020` or `2010-12`. To look up the format of a statistical variable, see below.
+- `""`: Return observations for all dates. 
+
+#### Find the date format for a statistical variable
+
+Statistical variable dates are defined as yearly, monthly, weekly, or daily. For most variables, you can find out the correct date format by searching for the variable in the
+[Statistical Variable Explorer](https://datacommons.org/tools/statvar) and looking for the **Date range**. For example, for the variable [Gini Index of Economic Activity](https://datacommons.org/tools/statvar#sv=GiniIndex_EconomicActivity), the date-time format is yearly, i.e. in YYYY format:
+
+![date time example 1](/assets/images/rest/date_time_example1.png){: width="900"}
+
+For other cases, you may need to drill down further to a timeline graph to view specific observations. For example, [Mean Wind Direction](https://datacommons.org/tools/statvar#sv=Mean_WindDirection), is measured at the sub-daily level, but the frequency is not clear (hourly or every two hours, etc.)
+
+![date time example 2](/assets/images/rest/date_time_example2.png){: width="900"}
+
+In these cases, do the following:
+
+1. In the Statistical Variable Explorer, in click on an example place to link to the variable's page in the Knowledge Graph Browser. 
+1. Scroll to the **Observations** section and click **Show Table** to get a list of observations.
+
+For example, and click on the line graph to see the frequency, in the case of Mean Wind Direction for [Ibrahimpur, India](https://datacommons.org/browser/cpcbAq/636_Ibrahimpur_Vijayapura?statVar=Mean_WindDirection), the observations table shows that the variable is measured every four hours, starting at midnight.
```

Comment:
this sentence is slightly confusing to me .. the "and click on the line graph to see the frequency" feels out of place?

Reply:
Something got bungled here. "and click...frequency" should just be deleted. Fixed now.